lib/pusher/client.rb

```diff
@@ -88,6 +98,10 @@ def encrypted?
       @scheme == 'https'
     end
 
+    def cluster=(cluster)
+      @host = "api-#{cluster}.pusher.com"
```

Comment:
The `@port` should also be reset to `nil` in that case. We then depend on the `@scheme` to select the right port.